src/OutputWriters/windowed_time_average.jl

```diff
@@ -266,22 +310,25 @@ function advance_time_average!(wta::WindowedTimeAverage, model)
         wta.window_start_time = model.clock.time
         wta.window_start_iteration = model.clock.iteration
         wta.previous_collection_time = model.clock.time
+        # print("elseif end_of_window has been triggered! Setting collecting = false")
```

Comment:
@liuchihl mentioned that the logic breaks down if the interval is equal to a single timestep, because the logic flow only goes through `elseif` statement and never has to chance to accumulate.

Can we address this corner case by include the following in here:

```
if end_of_window(wta.schedule, model.clock)
  accumulate_result!(wta, model)
```